pyprusalink/__init__.py

```diff
@@ -40,33 +40,33 @@ async def resume_job(self, jobId: int) -> None:
     async def get_version(self) -> VersionInfo:
         """Get the version."""
         async with self.client.request("GET", "/api/version") as response:
-            return await response.json()
+            return response.json()
 
     async def get_legacy_printer(self) -> LegacyPrinterStatus:
         """Get the legacy printer endpoint."""
         async with self.client.request("GET", "/api/printer") as response:
-            return await response.json()
+            return response.json()
 
     async def get_info(self) -> PrinterInfo:
         """Get the printer."""
         async with self.client.request("GET", "/api/v1/info") as response:
-            return await response.json()
+            return response.json()
 
     async def get_status(self) -> PrinterStatus:
         """Get the printer."""
         async with self.client.request("GET", "/api/v1/status") as response:
-            return await response.json()
+            return response.json()
 
     async def get_job(self) -> JobInfo:
         """Get current job."""
         async with self.client.request("GET", "/api/v1/job") as response:
             # when there is no job running we'll an empty document that will fail to parse
             if response.status == 204:
```

Comment:
This needs to be changed as well. I got: AttributeError: 'Response' object has no attribute 'status'

Reply:
Thanks, I missed that.